.github/workflows/BuildAndTest.yml

```diff
@@ -1,33 +1,46 @@
-# This workflow will do a clean install of node dependencies, cache/restore them, build the source code, use linter to check the code and run tests
-# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+name: Build and test
 
-name: Build, lint and test
+# Variables
+env:
+  BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
 
+# Triggering the workflow on PRs and pushes to relevant branches
 on:
   pull_request:
     branches:
       - main
+      - feature/**
   push:
```

Comment:
We don't need to run this on main (once merged)

Reply:
Removed on `push` event